.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2.1
```

Comment:
Can we not use GitHub actions for CI instead of CircleCI? This is a genuine question, but I was hoping we could migrate away from CircleCI now that GH Actions are OK to use.

Reply:
yep, you've already added the secrets for it via your [Terraform PR](https://github.com/dwp/dataworks-github-config/pull/97). Examples here: https://github.com/dwp/ucfs-claimant-api-load-data/tree/master/.github/workflows

Reply:
Thanks Matt , Dan . I have removed .circleci now and add .github